### PR TITLE
update travis jdk8 build

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,9 @@ sudo: false
 language: java
 matrix:
   include:
-    - name: Oracle JDK 8
+    - name: Oracle JDK 8 on Trusty
```
Comment (loosebazooka) [inline .travis.yml]: see jib: https://github.com/GoogleContainerTools/jib/pull/1874

Comment (chanseokoh) [reply .travis.yml]: I think we should prefer `openjdk8` if it works. In the Jib case, `openjdk8` has a Javadoc-related issue.

Comment (loosebazooka) [reply .travis.yml]: I think we ran into the same issue with javadoc, that's why we use openjdk, but I don't remember exactly.

Comment (loosebazooka) [reply .travis.yml]: let me try.

**Comment by loosebazooka** [comment]
closed in favor of #397

**Pull request closed without merging**